### PR TITLE
[CORE-1715] pachctl inspect defaults --cluster

### PR DESCRIPTION
**Pull request by robert-uhl**
- Implement `pachctl inspect defaults --cluster`.
- Also merge in the latest master branch.

### CODE REVIEW

File: .circleci/main.yml [hunk 1]
```diff
@@ -119,7 +119,7 @@ jobs:
     environment:
       TEST_RESULTS: /tmp/test-results
       GOPROXY: https://proxy.golang.org
-    parallelism: 8
+    parallelism: 10
```
Comment (molinamelendezj) [inline .circleci/main.yml:122]: I think @djanicekpach has a PR up for this. one you might want to revert ?

File: src/internal/clusterstate/v2.5.0/collection.go [hunk 2]
```diff
@@ -280,6 +286,8 @@ func migratePostgreSQLCollection(ctx context.Context, tx *pachsql.Tx, name strin
 				query := fmt.Sprintf("update collections.%s set %s where key = :oldKey", col.table, strings.Join(updateFields, ", "))
 
 				_, err = col.tx.NamedExecContext(ctx, query, params)
```
Comment (molinamelendezj) [inline src/internal/clusterstate/v2.5.0/collection.go:288]: should we check this error since we have `i++`? or is that what `col.mapSQLError()` is for?

**Review by molinamelendezj — APPROVED** [review]
few minor comments by otherwise LGTM

File: .circleci/main.yml (hunk 1 above)
Comment (robert-uhl) [reply .circleci/main.yml:122]: This is the latest committed version (I merged master into this branch after making my own changes).

File: src/internal/clusterstate/v2.5.0/collection.go (hunk 2 above)
Comment (robert-uhl) [reply src/internal/clusterstate/v2.5.0/collection.go:288]: That’s what the `col.mapSQLError()` is for, as you guessed — it’s from my merge of master into this branch.

**Pull request merged**